includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -262,21 +274,29 @@ public function get_product_data() {
 			}
 		}
 
-		$data          = [];
-		$items         = [];
-		$product_price = $this->get_product_price( $product );
+		try {
+			$price = $this->get_product_price( $product );
```

Comment:
Any specific reason to rename it from `product_price`?

Reply:
It's just a personal nitpick of mine. If there is a single price in the method, I prefer not to be overly-specific and choose the shorter version. If it was a single instance, and I didn't have changes within the method, I would have left it alone, but decided to use the opportunity :)